src/main/java/com/mongodb/kafka/connect/util/SslConfigs.java

```diff
@@ -0,0 +1,186 @@
+            Importance.MEDIUM,
+            CONNECTION_SSL_KEYSTORE_PASSWORD_DOC,
+            group,
+            ++orderInGroup,
+            Width.MEDIUM,
+            CONNECTION_SSL_KEYSTORE_PASSWORD_DISPLAY);
+    return configDef;
+  }
+
+  /**
+   * Set key store and trust store parameters
+   *
+   * @param sslSettingsBuilder - SSL Context Builder from MogoClient
+   * @param config - Sink our Source Connector properties with key/trust store parameters
+   */
+  public static void setupSsl(
+      final SslSettings.Builder sslSettingsBuilder, final AbstractConfig config) {
+    try {
+      TrustManager[] trustManagers = null;
+      KeyManager[] keyManagers = null;
+
+      // trust store configuration should be applied only if supplied:
+      String storePath = config.getString(CONNECTION_SSL_TRUSTSTORE_CONFIG);
+      Password storePassword = config.getPassword(CONNECTION_SSL_TRUSTSTORE_PASSWORD_CONFIG);
+
+      storePath = storePath != null ? storePath.trim() : null;
+      if (storePath != null && !storePath.isEmpty()) {
+        try (InputStream trustStoreInputStream = Files.newInputStream(Paths.get(storePath))) {
+
+          KeyStore trustStore = KeyStore.getInstance(KeyStore.getDefaultType());
+          trustStore.load(trustStoreInputStream, storePassword.value().trim().toCharArray());
+
+          TrustManagerFactory trustManagerFactory =
+              TrustManagerFactory.getInstance(TrustManagerFactory.getDefaultAlgorithm());
+          trustManagerFactory.init(trustStore);
+          trustManagers = trustManagerFactory.getTrustManagers();
+        }
+      }
+
+      // let's do the same to the key store configuration:
+      storePath = config.getString(CONNECTION_SSL_KEYSTORE_CONFIG);
+      storePassword = config.getPassword(CONNECTION_SSL_KEYSTORE_PASSWORD_CONFIG);
+
+      storePath = storePath != null ? storePath.trim() : null;
+      if (storePath != null && !storePath.isEmpty()) {
+        try (InputStream keyStoreInputStream = Files.newInputStream(Paths.get(storePath))) {
+
+          char[] pwd = storePassword.value().trim().toCharArray();
+          KeyStore keyStore = KeyStore.getInstance(KeyStore.getDefaultType());
+          keyStore.load(keyStoreInputStream, pwd);
+
+          KeyManagerFactory keyManagerFactory =
+              KeyManagerFactory.getInstance(KeyManagerFactory.getDefaultAlgorithm());
+          keyManagerFactory.init(keyStore, pwd);
+          keyManagers = keyManagerFactory.getKeyManagers();
+        }
+      }
+
+      if (trustManagers == null && keyManagers == null) {
+        // either key or trust managers can be null, and SSLConext can be updated with nulls
+        // in this case the installed security providers will
+        // be searched for the highest priority implementation of the appropriate factory.
+        // But, if SSL configuration parameters aren't provided (both are null),
+        // it's better to leave SSLConext without a change at all.
```

Comment:
SSLConext => SSLContext

Reply:
Fixed the typo.
thanks for catching this... 👍